register.js

```diff
@@ -0,0 +1,54 @@
+
+
+document.getElementById('submitButton').addEventListener('click', (e) => {
+    //Prevents default form action from happening
+    e.preventDefault();
```

Comment:
Why did you use this method, and what does it do?

Reply:
when i first tried to run the web page the results were directly disappearing, to solve this problem i used prevent defaults which cancels the event if it is able to be canceled, meaning that the default action that belongs to the event will not occur.

Reply:
Cool